cet/quick_test.c

```diff
@@ -13,6 +13,13 @@
 #include <sys/wait.h>
 #include <sched.h>
 
+#define get_ssp()						\
+({								\
+	unsigned long _ret;						\
```

Comment:
It should be better if the '\\' is aligned.

Reply:
Yes, done.

Thanks!